internal/component/database_observability/mysql/collector/parser/parser.go

```diff
@@ -1,23 +1,16 @@
 package parser
 
-import "github.com/go-kit/log"
+import (
+	"github.com/pingcap/tidb/pkg/parser/ast"
+)
 
 type Parser interface {
```

Comment:
Do we still need the interface?

Reply:
It's useful because of the mock parser we have in some tests.